react/FilePicker/queries.js

```diff
@@ -0,0 +1,36 @@
+import { Q, fetchPolicies } from 'cozy-client'
+
+const FILES_DOCTYPE = 'io.cozy.files'
+
+const defaultFetchPolicy = fetchPolicies.olderThan(30 * 1000)
+const TRASH_DIR_ID = `${FILES_DOCTYPE}.trash-dir`
+
+export const buildCurrentFolderQuery = folderId => ({
+  definition: () => Q(FILES_DOCTYPE).getById(folderId),
+  options: {
+    as: `buildCurrentFolderQuery-${folderId}`,
```

Comment:
Let's name it as Drive in order to share the same state? 
    `as: 'onlyfolder-' + folderId`